include/swift/AST/LookupKinds.h

```diff
@@ -65,8 +65,14 @@ enum NLOptions : unsigned {
   /// primary source file.
   ///
   /// \see NL_KnownDependencyMask
-  NL_KnownNoDependency =
+
+  // Work around MSVC bug: expression did not evaluate to a constant.
```

Comment:
If you're going to do this, just write the constant unconditionally, and static_assert that it has the correct value.